src/main/java/at/ac/tuwien/kr/alpha/grounder/NaiveGrounder.java

```diff
@@ -575,6 +576,16 @@ public boolean isFact(Atom atom) {
 		return analyzeUnjustified.analyze(atomToJustify, currentAssignment);
 	}
 
+	@Override
+	public <E extends Assignment.Entry> Integer getSomeMBTAssignedAtom(Assignment<E> assignment) {
```

Comment:
Why does this return `Integer` and not `int`?

Reply:
TBH I do not know. This method was moved from `ArrayAssignment` to here, the signature is by @AntoniusW